README.md

```diff
@@ -1,18 +1,24 @@
 ## TDR Generated GraphQL
 
-This project generates, packages and deploys to the sonatype nexus the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project.
+This project generates, packages and deploys the case classes generated by the [sbt-graphql](https://github.com/muuki88/sbt-graphql) project to the sonatype nexus.
 
-These classes will be used by other TDR repositories to communicate with the consignment API.
+These classes are be used by other TDR repositories to communicate with the consignment API.
 
 ### Building locally
-* Add a new query file to `src/main/graphql`
+* If you add a new query file to `src/main/graphql`:
 * Add a schema file generated by the [consignment api](https://github.com/nationalarchives/tdr-consignment-api) project. This will be generated in the `target/sbt-graphql` directory and should be copied into `src/main/resources/schema.graphql`
-* Run `sbt package publishLocal`
+* Run `sbt package publishLocal` - This allows you to use your updated version of the API locally to facilitate work on FE (This helps as sometimes Jenkins can be slow to build and catch up etc.)
```

Comment:
The main reason to do this is because you want to check the changes work locally before you get them reviewed and published, rather than because of Jenkins. By the way, I think it's Sonatype that's the slow part. @MancunianSam might know more.

Reply:
You're right, it is because you want to check your work locally first and yes, it's the Sonatype deploy that takes the time.

Reply:
Cool, I'll change that to reflect this!